docs/about/frequently-asked-questions.mdx

```diff
@@ -0,0 +1,138 @@
+7. Modular: every block/cell you write is a standalone file that is interoperable; meaning it can be used in other pipelines or in other code bases.
+
+</Accordion>
+
+<Accordion title ="What’s the difference between Mage and Fivetran?">
+
+Check out our blog [Mage vs. Fivetran](https://www.mage.ai/blog/mage-vs-fivetran).
+
+</Accordion>
+
+<Accordion title ="What’s the difference between Mage and Airbyte?">
+
+Check out our blog [Mage vs Airbyte](https://www.mage.ai/blog/mage-vs-airbyte).
+
+</Accordion>
+
+<Accordion title ="What’s the difference between Mage and Prefect?">
+
+Mage provides an interactive notebook with built-in engineering best practices for building pipelines, which makes prototyping and building production-ready pipelines much easier.
+
+Mage supports writing pipelines in multiple languages which include Python, SQL, and R.
+
+Mage supports multiple types of pipelines natively such as:
+
+- Standard batch pipelines
+- Data integration pipelines
+- Streaming pipelines
+- Spark pipelines
+- DBT pipelines
+
+</Accordion>
+
+<Accordion title ="What languages does Mage support?">
+
+We currently support SQL, Python, R, and PySpark. Coming soon: Spark SQL.
+
+</Accordion>
+
+<Accordion title ="Does Mage integrate with Spark?">
+
+Yes! [Here](https://docs.mage.ai/integrations/spark-pyspark) is a step-by-step tutorial to use Mage with Spark on EMR.
+
+</Accordion>
+
+<Accordion title ="What’s the difference between Mage and Sagemaker?">
+
+Sagemaker is used to train machine learning models and serve them via api.
+
+Mage is an engine for running data pipelines that can move and transform data. That data can then be stored anywhere (e.g. S3) and used to train models in Sagemaker.
+
+</Accordion>
+
+<Accordion title ="What’s the difference between Mage and Databricks?">
+
+Databricks provides infrastructure to run Spark. They also provide notebooks that can run your code in Spark as well.
+
+Mage can execute your code in a Spark cluster, managed by AWS, GCP, or even Databricks.
+
+</Accordion>
+
+<Accordion title ="How do I send pipeline notifications to Slack?">
+
+[Here is a doc](/production/observability/alerting-slack) to help you set up alerting for pipeline status updates in [Slack](https://www.mage.ai/chat).
+
+</Accordion>
+
+<Accordion title ="How can I contribute or request features?">
+
+We love and welcome community contributions! [Here is a doc](/community/contributing) to get you started.
+
+To request features, add a “Feature request” using the `New issue` button in Github from this [link](https://github.com/mage-ai/mage-ai/issues), or join our [feature-request](https://www.mage.ai/chat) Slack channel.
+
+</Accordion>
+</AccordionGroup>
+
+*Can’t find what you’re looking for? [Ask a question here](https://github.com/mage-ai/mage-ai/discussions/categories/q-a) or [join our slack](https://mage.ai/chat) for additional support!*
```

Comment:
@mattppal if we going to disable the discussions, we'll need to update the link here.